department-of-veterans-affairs/caseflow
# Add VHA Regional Office (VISN) team management page section

### Description
Puts VHA Regional Offices (VISNs) into their own section on the Team Management page.

### Acceptance Criteria

### Testing Plan

1. Run VHA Regional Office DB seed: https://github.com/department-of-veterans-affairs/caseflow/pull/16653
2. Go to /team_management
3. Confirm VHA Regional Offices (VISNs) appear in their own section.

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1761" alt="Screen Shot 2021-08-24 at 12 05 25 PM" src="https://user-images.githubusercontent.com/640182/130651189-1850b4a2-c3e4-48a2-81fe-7d1923579451.png">|<img width="1778" alt="Screen Shot 2021-08-24 at 12 18 58 PM" src="https://user-images.githubusercontent.com/640182/130653140-50f50000-5b7d-4856-97f6-bf601a50b856.png">
